.github/workflows/release.yml

```diff
@@ -19,7 +20,27 @@ on:
       - '*'
 
 jobs:
-  Build-Release:
+  Package-Source-Release:
+    strategy:
+      matrix:
+        os: [ubuntu-20.04, windows-2022]
```

Comment:
Only difference here is that one package is packaged as .tar.gz and the other as .zip. Inside they should be the same.

Reply:
So these produce the same source distribution, just one is zip and the other tar.gz . Honestly I could produce both of them in one step, I just thought that this would also test packaging logic for windows, even if it might not be necessary,

Reply:
We can keep things simple and start with only shipping `tar.gz` source bundles. [Windows ships with `tar` since some time.](https://learn.microsoft.com/en-us/virtualization/community/team-blog/2017/20171219-tar-and-curl-come-to-windows)

Reply:
windows build (the zip generator) disabled